blueprint.md

```diff
@@ -115,6 +115,12 @@ Normally an [`anchor element`](https://developer.mozilla.org/en-US/docs/Web/HTML
 
 There are many advantages of using an anchor element, the main one being accessibility.
 
+Alternatively, if you would still like to allow relative links to other parts of your site to navigate as normally, you can opt out of this behavior on a link-by-link basis:
+
+```html
+<a href="/about" data-router-slot="disable">Go to about!</a>
```

Comment:
If you also change the value of the attribute to `disabled` here everything looks good 👍